### ISSUE TITLE
send key in plugin

### ISSUE THREAD

**Issue by fabio1999ita** [soarqin/D2RMH#87]
is possible in lua send key of keyboard?

**Comment by soarqin** [comment]
yeah, I have to add map/monster/object data querying interface to script first, and input functions after that

**Comment by fabio1999ita** [comment]
i wanted to add the keystroke to cast skill
